Slider/index.js

```diff
@@ -0,0 +1,118 @@
+var _extends =
+  Object.assign ||
+  function(target) {
+    for (var i = 1; i < arguments.length; i++) {
+      var source = arguments[i];
+      for (var key in source) {
+        if (Object.prototype.hasOwnProperty.call(source, key)) {
+          target[key] = source[key];
+        }
+      }
+    }
+    return target;
+  };
+
+function _objectWithoutProperties(obj, keys) {
+  var target = {};
+  for (var i in obj) {
+    if (keys.indexOf(i) >= 0) continue;
+    if (!Object.prototype.hasOwnProperty.call(obj, i)) continue;
+    target[i] = obj[i];
+  }
+  return target;
+}
+
+import { h } from "preact";
+import MaterialComponent from "../MaterialComponent";
+import { MDCSlider } from "@material/slider";
+
+/**
+ * @prop disabled = false
+ */
+export default class Slider extends MaterialComponent {
+  constructor() {
+    super();
+    this.componentName = "slider";
+    this._mdcProps = ["disabled", "min", "max", "value", "step"];
```

Comment:
So the way _mdcProps works is:
if user gives the value as true/false then these will be converted to specific `mdc_componentname--classname`

https://github.com/prateekbh/preact-material-components/blob/master/MaterialComponent.js#L35-L37

So I dont this these are needed here